_layouts/base.html.haml

```diff
@@ -24,6 +24,7 @@ change_frequency: monthly
     %link(rel="stylesheet"){:href => relative("/website/highlight/highlight.css")}
     %link(rel="stylesheet"){:href => relative("/website/tabzilla/tabzilla.css")}
     %link(rel="stylesheet"){:href => relative("/website/optaplannerWebsite.css")}
+    %link(rel="stylesheet"){:href => "https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css"}
```

Comment:
I agree, see PR comment ^^ with the issue I had copying their files to our website folder.